### DISCUSSION TITLE
Is there any way to reset local offset on every beatmap?

### DISCUSSION THREAD

**Discussion by okolenmi** [ppy/osu#24324] [Q&A]
Not so long ago I changed my keyboard and felt that the offsets that I set no longer fit. I set them on a very large number of maps and now I'm facing a problem when I play such a map in multiplayer, I press keys too early because of the local offset. Is it possible to reset offset for all maps at the same time?
I know about global offset but, really, mappers sometimes can't set offset correctly (+-15ms mistakes). So earlier I decided to set it manually on every map after playing it. It would be possible to solve the problem by shifting the global offset, but I did not think and started to reconfigure it on each map manually. Now I have a mess with good enough and bad offsets.

**Comment by peppy** [comment]
The easiest way is probably to upload your `client.realm` (you can compress it and drop in this issue thread) and we'll reset them for you.
